test/e2e/policyrecommendation_test.go

```diff
@@ -172,51 +171,51 @@ func TestPolicyRecommendation(t *testing.T) {
 
 // Example output: Successfully created policy recommendation job with ID e998433e-accb-4888-9fc8-06563f073e86
 func testPolicyRecommendationRun(t *testing.T, data *TestData) {
-	stdout, jobId, err := runJob(t, data)
+	stdout, jobName, err := runJob(t, data)
 	require.NoError(t, err)
 	assert := assert.New(t)
-	assert.Containsf(stdout, fmt.Sprintf("Successfully created policy recommendation job with ID %s", jobId), "stdout: %s", stdout)
+	assert.Containsf(stdout, fmt.Sprintf("Successfully created policy recommendation job with name %s", jobName), "stdout: %s", stdout)
 }
 
 // Example output: Status of this policy recommendation job is COMPLETED
 func testPolicyRecommendationStatus(t *testing.T, data *TestData) {
-	_, jobId, err := runJob(t, data)
+	_, jobName, err := runJob(t, data)
 	require.NoError(t, err)
-	stdout, err := getJobStatus(t, data, jobId)
+	stdout, err := getJobStatus(t, data, jobName)
 	require.NoError(t, err)
 	assert := assert.New(t)
 	assert.Containsf(stdout, "Status of this policy recommendation job is", "stdout: %s", stdout)
 }
 
 // Example output:
-// CreationTime          CompletionTime        ID                                   Status
-// 2022-06-17 15:03:24 N/A                 615026a0-1856-4107-87d9-08f7d69819ae RUNNING
+// CreationTime          CompletionTime        Name                                   Status
+// 2022-06-17 15:03:24 N/A                 615026a0-1856-4107-87d9-08f7d69819ae RUNNIN
```

Comment:
Ditto, missing `pr-` prefix of Name column

Reply:
Fixed.